spec/lib/que/locker_spec.rb

```diff
@@ -94,8 +94,11 @@ def expect_to_lock_with(cursor:)
         context "with non-zero cursor expiry" do
           let(:cursor_expiry) { 5 }
 
-          before { allow(locker).to receive(:monotonic_now) { @epoch } }
-
+          before do
+            # we need this to avoid flakiness during resetting the cursor
```

Comment:
Talked IRL, @ankithads to explain in the comment why this is needed

Reply:
updated